src/sockets/websocket.py

```diff
@@ -111,10 +111,18 @@ def handleNameStagingState(self, user, str):
     def handleReadyState(self, user, str):
         #check existence of name and message
         if not ProcessText.hasNameandMessage(str):
-            self.write_message("who is the recipient and what is the message")
+            if ProcessText.hasGreetings(str):
```

Comment:
can be simplified to 
not ProcessText.hasNameandMessage(str) and ProcessText.hasGreetings(str)

Reply:
at that point, it should be extracted to a named function